app/services/catalog/portfolio_permission.rb

```diff
@@ -3,8 +3,6 @@ class PortfolioPermission
     VALID_RESOURCE_VERBS = %w[read write order].freeze
 
     def self.verify_permissions(values)
-      raise Catalog::InvalidParameter, 'Permission should be an array' unless values.kind_of?(Array)
```

Comment:
Is this check no longer needed?  Could `permission_array_check` be used here instead?

Reply:
Spoke in person with Drew, moved this logic into the RBACMixin as well since that is where the validation belongs.